### PR TITLE
[Wallet] Withdraw art

### LINKED ISSUE #4921: Add and Withdraw Screen missing artwork
### Expected Behavior

Should have artwork: [Figma](https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet-Refresh?node-id=8284%3A29)

Cash In and Out Landing

### Current Behavior

Artwork missing from Add/Withdraw landing screen.

### PR DESCRIPTION
**Pull request by medhakothari**
### Description

Add and Withdraw Screen (aka FiatExchange) was missing artwork, so that was added. 

### Tested

- Tested on multiple phones, 2 of their screenshots attached:
Simulator Screen Shot - iPhone 8 - 2020-09-10 at 13 37 15 
Screen Shot 2020-09-10 at 2 27 41 PM

### Related issues

- Fixes #4921 
### Backwards compatibility

n/a

### CODE REVIEW

**Comment by jmrossy** [comment]
@medhak1 I think the art needs to be added to the private visual asset repo. @jeanregisser can confirm.

File: packages/mobile/src/account/FiatExchange.tsx [hunk 1]
```diff
@@ -82,8 +86,13 @@ function FiatExchange() {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
+    justifyContent: 'center',
+  },
+  image: {
+    // height: 200,
+    // alignItems: 'center',
```
Comment (annakaz) [inline packages/mobile/src/account/FiatExchange.tsx]: Can these be removed?

**Review by annakaz — COMMENTED** [review]
Thanks for adding this, I think @jeanregisser has instructions on how to add encrypted assets 
https://github.com/celo-org/celo-monorepo/tree/master/packages/mobile/branding/celo/src/images just contains placeholders for security reasons

**Comment by medhakothari** [comment]
Got it, just added a pink placeholder image to the monorepo. Should be fine now! @jmrossy @annakaz

**Review by jeanregisser — APPROVED** [review]
Looking great! 👍

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine  i have verified task using Test flight build v1.2.0 (24) and found the following:

**Test scenarios:**

- Verify user is able to see the Add & Withdraw button when tapping on hamburger menu 
- Verify user is able to see the artwork on Add & withdraw page


Please let me know if any thing else i need to verify for this task.
Thanks !

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine i have verified task using  Android internal build 1.2.0(1004294313) and found the following
Device: Samsung Galaxy A70(9.0)
Test scenarios:

- Verify user is able to see the Add & Withdraw button when tapping on hamburger menu
- Verify user is able to see the artwork on Add & withdraw page

Please let me know if any thing else i need to verify for this task.
Thanks !

**Pull request merged**